core/commons/che-core-commons-test/pom.xml

```diff
@@ -21,10 +21,30 @@
     <artifactId>che-core-commons-test</artifactId>
     <packaging>jar</packaging>
     <name>Che Core :: Commons :: Utilities for tests</name>
+    <properties>
+        <findbugs.failonerror>true</findbugs.failonerror>
+    </properties>
     <dependencies>
         <dependency>
+            <groupId>com.google.inject</groupId>
+            <artifactId>guice</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>javax.inject</groupId>
+            <artifactId>javax.inject</artifactId>
```

Comment:
yes
https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope
provided
_This is much like compile, but indicates you expect the JDK or a container to provide the dependency at runtime. For example, when building a web application for the Java Enterprise Edition, you would set the dependency on the Servlet API and related Java EE APIs to scope provided because the web container provides those classes. This scope is only available on the compilation and test classpath, and is not transitive._


so to sum up, in Eclipse Che javax.inject, guice are provided by Eclipse Che framework as it's "core stuff" to every server component (the platform) so all dependencies should use provided on these "platform depedencies"